### ISSUE TITLE
Chart range in combination with chart timestamp lead to strange behaviour

### ISSUE THREAD

**Issue by lukasholzer** [dynatrace-oss/barista#96]
In our integration of the chart if we enable both, the timestamp and the range selection, we experience strange behaviour, as described below:

Reproduction:
 - Select a chart range
 - Select a chart timestamp
 - Select a range again
 - Scroll up and down

We then end up with the overlay showing \{from} - \{to} and a console error:
```
[ERROR] DtDateRange Formatter | The DtDateRange formatter expects an array of two numbers that should be formatted!
```
Could you please check if this is a problem inside the component or the integration.

**Comment by ffriedl89** [comment]
@lukasholzer Is this still relevant?

**Comment by lukasholzer** [comment]
This should be fixed with #432.

**Comment by lukasholzer** [comment]
@peter-affenzeller maybe you can create one e2e testing scenario that reflects this reproduction.
